OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -277,33 +306,44 @@ OpenSim::Device* createDevice() {
     // Add sphere (geometry) attach them to the cuffs
     sphere.setFrameName("cuffA");
     cuffA->addGeometry(sphere);
-    // TODO: cuffB
+    sphere.setFrameName("cuffB");
+    cuffB->addGeometry(sphere);
 
     // Joint from something in the environment to cuffA.
     // It will be used to attach the device at cuffA to a model.
     auto anchorA = new OpenSim::WeldJoint();
-    // TODO: set name
-
+    anchorA->setName("anchorA");
     // Set only the child now. Parent will be in the environment.
-    // TODO
-
+    anchorA->setChildFrameName("cuffA");
+    device->addComponent(anchorA);
 
     // Joint from something in the environment to cuffB.
     // It will be used to attach the device at cuffA to a model.
-    // TODO
+    auto anchorB = new OpenSim::WeldJoint();
+    anchorB->setName("anchorB");
+    // Set only the child now. Parent will be in the environment.
+    anchorB->setChildFrameName("cuffB");
+    device->addComponent(anchorB);
 
-    // PathActuator connecting the two cuffs (A and B).
-    // TODO
+    // Actuator connecting the two cuffs (A and B).
+    auto pathActuator = new OpenSim::PathActuator();
+    pathActuator->setName("cableAtoB");
+    pathActuator->set_optimal_force(OPTIMAL_FORCE);
+    pathActuator->addNewPathPoint("point1", *cuffA, Vec3(0));
+    pathActuator->addNewPathPoint("point2", *cuffB, Vec3(0));
+    device->addComponent(pathActuator);
 
     // A controller that specifies the control to the actuator
-    // TODO
-    // TODO: finish implementing the PropMyoController class, above.
+    auto controller = new OpenSim::PropMyoController();
+    controller->setName("controller");
+    controller->set_gain(GAIN);
 
     // Connect the the controller to the device actuator
```

Comment:
Whoops, I did not mean to commit my local changes to the example! Ignore these, Sorry @tkuchida. I will close and make a new PR.